### ISSUE TITLE
try out weighting PSL in sample

### ISSUE THREAD

**Issue by bogsnork** [naturalengland/EPM_samp_strat#34]

**Comment by bogsnork** [comment]
Can probably do this with the `eta` parameter.  Need to create a matrix of weights.  From `clhs` documentation: 
```
df <- data.frame(
a = runif(1000),
b = rnorm(1000),
c = sample(LETTERS[1:5], size = 1000, replace = TRUE)
)

# Method DLHS with a linear increase of the strata weight (i.e. probability to be sampled)
# from 1 for the middle starta to 3 for the edge of the distribution
linear_increase <- 1+(2/24)*0:24
eta <- matrix(c(rev(linear_increase), linear_increase), ncol = 2, nrow = 50)
set.seed(1)
res <- clhs(df, size = 50, iter = 100, eta = eta, progress = FALSE, simple = FALSE)
str(res)
plot(res)
```

**Comment by bogsnork** [comment]
the above example is a bit weird in how it defines 'edge' of distribution - assumes the values are ordered somehow.

**Comment by bogsnork** [comment]
done and closed with #39
